internal/packages/packageloader/transformfiles_funcs.go

```diff
@@ -0,0 +1,258 @@
+package packageloader
+
+import (
+	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
+)
+
+// allow all sprig functions except os, network and filepath.
+var allowedFuncNames = map[string]struct{}{
+	"hello": {},
+
+	// Dates
+	"ago":              {},
+	"date":             {},
+	"date_in_zone":     {},
+	"date_modify":      {},
+	"dateInZone":       {},
+	"dateModify":       {},
+	"duration":         {},
+	"durationRound":    {},
+	"htmlDate":         {},
+	"htmlDateInZone":   {},
+	"must_date_modify": {},
+	"mustDateModify":   {},
+	"mustToDate":       {},
+	"now":              {},
+	"toDate":           {},
+	"unixEpoch":        {},
```

Comment:
Please drop `now` `unixEpoch` `html*` `ago`  and what's with the lower snake cased function names?
Is it `date_in_zone` or `dateInZone`? The documentation only mentions the latter.

Reply:
see [here](https://github.com/package-operator/package-operator/pull/172#issuecomment-1431244223)